### PR TITLE
Addition of Bio for Beverley

### PR DESCRIPTION
**Pull request by BNewing**
Added in the text into the team page file and also added in a photo in the photos folder. 
@camallen @simoneduca

### CODE REVIEW

**Comment by camallen** [comment]
@BNewing can you shrink your bio image to match the other images in `./public/assets/team/` ? I think it's ~ 150px maintaining aspect ratio..

File: app/pages/about/team-page.cjsx [hunk 1]
```diff
@@ -187,6 +190,13 @@ teamMembers =
     bio: counterpart "team.content.andreaSimenstad.bio"
     image: "/assets/team/andrea.jpg"
     location: "minnesota"
+  beverleyNewing:
+    name: "Beverley Newing"
+    twitter: "beverleynewing"
+    title: counterpart "team.content.beverleyNewing.title"
+    bio: counterpart "team.content.beverleyNewing.bio"
+    image: "/assets/team/bev.jpg"
```
Comment (camallen) [inline app/pages/about/team-page.cjsx]: can you make this filename a little more explicit, i.e. `beverley.jpg`. I read that with more information...naming things is hard.

**Comment by camallen** [comment]
some minor comments. looking good.

**Comment by BNewing** [comment]
Ah! I've changed the photo to 150px square and changed the file name from bev to beverley and pushed it to the branch @camallen

**Comment by camallen** [comment]
seems both files exist and it's using the old one still.

**Comment by camallen** [comment]
LGTM

**Comment by BNewing** [comment]
@camallen

**Comment by alexbfree** [comment]
The images are actually displayed at 80px x 80px not 150px x 150px . But so long as it's square that's fine, it gets resized. I notice some of the others are also getting auto-resized from a larger square. 

So, all good. Thanks Beverley! I will incorporate this into the team page updates I am doing with adding the alumni page etc.

**Comment by alexbfree** [comment]
Actually on merging this i've just noticed an issue. The image is added in the wrong place. It needs to be in `public/assets/team` not `dist/assets/team` (`dist` is a generated folder). Could you push an update to your branch and Cam or I can re-merge the PR for you?

**Pull request merged**